packages/bpk-component-datatable/src/bpk-data-table-column.scss

```diff
@@ -33,11 +42,30 @@
   overflow: hidden;
 }
 
-.bpk-data-table-column__sort-icon {
+.bpk-data-table-column__sort-icons {
   margin: 0 0 0 $bpk-spacing-xs;
-  fill: $bpk-color-gray-700;
 
   @include bpk-rtl {
     margin: 0 $bpk-spacing-xs 0 0;
   }
 }
+
+.bpk-data-table-column__sort-icon {
+  display: block;
+  fill: $bpk-color-gray-300;
+
+  &:hover {
+    fill: $bpk-color-gray-500;
+  }
+
+  &--selected {
+    &,
+    &:hover {
+      fill: $bpk-color-blue-700;
+    }
+  }
+}
+
+.bpk-data-table-column__sort-icon--up {
+  margin-bottom: -($bpk-border-size-xl * 2);
```

Comment:
Reusing a border mixin/bond doesn't really make sense here. What's the best way of solving this? Maybe there's a better approach (the purpose of this is to reduce the vertical spacing between the up and down icons).

Reply:
You could use `spacing-xs`, which is 6px, so you wouldn't have to do the `* 2` part. I can't really think of a better way to do it than how you have though, other than that.

Reply:
Sounds good - cheers :)